website/docs/reference/environments.md

```diff
@@ -8,112 +8,63 @@ title: Environments
-</div>
+Environments represent different stages in your development lifecycle. They allow you to manage your product releases from local development to production. [Projects](/reference/projects) and [feature flags](/reference/feature-toggles) are accessible in all environments, but each environment has different configurations. This allows you to enable a feature flag in development or test without enabling it in production.
 
-Environments are a way to organize activation strategy configurations for feature flags. A feature exists in all environments, as the goal is to release it as soon as possible, but it makes sense to configure activation differently depending on the environment. [Open Source](https://www.getunleash.io/pricing) and [Pro](https://www.getunleash.io/pricing) customers have two preconfigured environments, **development** and **production**. [Enterprise](https://www.getunleash.io/pricing) customers can have unlimited environments.
+The default environments are **development** and **production**. Unleash instances created before version `4.3` also have an environment called **default**. [Enterprise](https://www.getunleash.io/pricing) customers can create, clone, deprecate, and delete environments.
 
-Connected applications use environment-scoped API keys to ensure they download only the relevant feature flag configurations for the environment they are running in. Unleash tracks usage metrics per environment.
-
-![A graph showing how environments work. Each project can have multiple features, and each feature can have different activation strategies in each of its environments.](/img/environments_overview.svg 'A feature flag exists across all environments, but take different activation strategies per environment.')
+The set of all available environments is defined at the instance level. Additionally, each project can choose which of the environments are visible on the project level. The set of environments available to any given project is always a subset of the environments at the instance level. Each project must always have at least one active environment.
 
 ## Environment types
 
-All environments in Unleash have a **type**. When you create a new environment, you must also assign it a type.
-
-The built-in environment types are:
+Each environment has a name and one of the following types:
 - Development
 - Test
 - Pre-production
 - Production
 
-The **production** environment type is special: Unleash will show additional confirmation prompts when you change something that could impact users in environments of this type. The built-in "production" environment is a production-type environment.
-
-The other environment types do not currently have any functionality associated with them. This may change in the future.
+Environments of type production show additional confirmation prompts when making changes that may affect users. The default **production** environment is of type production.
```

Comment:
It also determines when a feature goes into the **live** lifecycle step. But not sure if we need to mention that here.

Reply:
Agreed, thanks! fabfdcc09a4d2f8e46049a4c7377791faab698c2

Reply:
Small note on the change in terms of feature lifecycle: 

* Live is reached for the feature when it is receiving metrics AND is exposed to users. In other words, some of the metrics have to report exposure.
* For pre-live it's enough to report metrics from any non-production environment

Reply:
thanks, I'll make that clarification